pkg/dmsclient/dmsclient.go

```diff
@@ -147,7 +157,7 @@ func (c *dmsClient) List(snitchToken string) (Snitch, error) {
 		return snitch, err
 	}
 
-	resp, err := c.do(req)
+	resp, err := c.do(req, "describe")
```

Comment:
`"describe"` made me raise an eyebrow as it doesn't match the name of the func... but it's actually the name of the func that's questionable, since we're retrieving a single resource. +1